packages/components/src/internal/components/administration/useAdministrationSubNav.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+import React, { useEffect } from 'react';
+
+import { useSubNavTabsContext } from '../navigation/hooks';
+
+import { AppURL } from '../../url/AppURL';
+import { useServerContext } from '../base/ServerContext';
+import { AUDIT_KEY } from '../../app/constants';
+import { isProjectContainer, isProductProjectsEnabled, isAppHomeFolder } from '../../app/utils';
+import { useContainerUser } from '../container/actions';
+import { ITab } from '../navigation/types';
+
+const PARENT_TAB: ITab = {
+    text: 'Dashboard',
+    url: AppURL.create('home'),
+};
+
+export const useAdministrationSubNav = (): void => {
+    const { clearNav, setNoun, setTabs } = useSubNavTabsContext();
+    const { container, moduleContext, user } = useServerContext();
+    const isAppHome = isAppHomeFolder(container, moduleContext);
+    const homeFolderPath = isAppHome ? container.path : container.parentPath;
+    const homeProjectContainer = useContainerUser(homeFolderPath, { includeStandardProperties: true });
+    const appHomeUser = homeProjectContainer?.user ?? user;
```

Comment:
`homeProjectContainer` will always be an object so no need for the defensive check.